### ISSUE TITLE
Mishandling of empty <mrow> in <munderover>?

### ISSUE THREAD

**Issue by dpvc** [zorkow/speech-rule-engine#142]
As reported on the [MathJax user forum](https://groups.google.com/forum/?fromgroups#!topic/mathjax-users/6a4kU5wG7Ok),

```
<math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"> 
  <munderover> 
    <mo class="MathClass-rel">→</mo>
    <mrow></mrow>
    <mo class="MathClass-op">≅</mo>
  </munderover> 
</math>
```

renders improperly as

```
<math xmlns="http://www.w3.org/1998/Math/MathML" display="inline" data-semantic-complexity="8">
  <munder data-semantic-type="underscore" data-semantic-role="arrow" data-semantic-id="4" data-semantic-children="3,1" data-semantic-embellished="relation" data-semantic-complexity="8">
    <mrow data-semantic-type="overscore" data-semantic-role="arrow" data-semantic-id="3" data-semantic-children="0,2" data-semantic-parent="4" data-semantic-embellished="relation" data-semantic-complexity="4">
      <mo class="MathClass-rel" data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="0" data-semantic-parent="3" data-semantic-complexity="1">&#x2192;</mo>
      <mo class="MathClass-op" data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-id="2" data-semantic-parent="3" data-semantic-complexity="1">&#x2245;</mo>
    </mrow>
    <mrow data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="1" data-semantic-parent="4" data-semantic-complexity="0" />
  </munder>
</math>
```

where the two `<mo>` elements are places together as the base of the `<munder>` and the empty row is placed below.

**Comment by zorkow** [comment]
That's seems to be an enrichment problem, as the semantic interpretation seems to be correct.

**Comment by pkra** [comment]
Filed downstream at https://github.com/mathjax/MathJax-a11y/issues/171

**Comment by zorkow** [comment]
Fixed in v2.0.0-beta.1.
